src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -220,26 +295,51 @@ def _run_single_channel(method: str, metadata: Sequence[Tuple[str, str]],
         _cancel_all_rpcs(futures)
 
 
+class _XdsUpdateClientConfigureServicer(
+        test_pb2_grpc.XdsUpdateClientConfigureServiceServicer):
+
+    def __init__(self, per_method_configs: Mapping[str, _ChannelConfiguration],
+                 qps: int):
+        super(_XdsUpdateClientConfigureServicer).__init__()
+        self._per_method_configs = per_method_configs
+        self._qps = qps
+
+    def Configure(self, request: messages_pb2.ClientConfigureRequest,
+                  context: grpc.ServicerContext
+                 ) -> messages_pb2.ClientConfigureResponse:
+        logging.info("Received Configure RPC: {}".format(request))
+        method_strs = (_METHOD_ENUM_TO_STR[t] for t in request.types)
+        for method in _SUPPORTED_METHODS:
+            method_enum = _METHOD_STR_TO_ENUM[method]
+            if method in method_strs:
+                qps = self._qps
+                metadata = ((md.key, md.value)
+                            for md in request.metadata
+                            if md.type == method_enum)
+            else:
+                qps = 0
+                metadata = ()
+            channel_config = self._per_method_configs[method]
+            with channel_config.condition:
+                channel_config.qps = qps
+                channel_config.metadata = list(metadata)
+                channel_config.condition.notify_all()
+        # TODO: Wait for all channels to respond until responding to RPC?
```

Comment:
It's ok, not need to enforce such a strong consistency. The test framework would give enough amount of time for test client to reach steady state.